saw-remote-api/Dockerfile

```diff
@@ -47,26 +47,24 @@ RUN useradd -m saw
 COPY --chown=saw:saw . /home/saw
 USER saw
 WORKDIR /home/saw
-ARG CRYPTOLPATH="/cryptol/.cryptol"
 ENV LANG=C.UTF-8 \
     LC_ALL=C.UTF-8
 COPY cabal.GHC-8.8.4.config cabal.project.freeze
-RUN mkdir -p rootfs/usr/local/bin
-RUN cabal v2-update && \
-    cabal v2-build -j saw-remote-api:exe:saw-remote-api && \
-    cp $(cabal v2-exec which saw-remote-api) rootfs/usr/local/bin
-RUN mkdir -p rootfs/"${CRYPTOLPATH}" \
-    && cp -r deps/cryptol/lib/* rootfs/"${CRYPTOLPATH}"
+RUN cabal v2-update
```

Comment:
The v2-update should be in the same RUN command to ensure that it's ran every time cabal v2-build is. Otherwise you might end up with a stale instance of cabal v2-update. (the v2-exec command can be in a separate RUN command though

Reply:
so instead of
```
RUN cabal v2-update
RUN cabal v2-build saw-remote-api
```
it should be
```
RUN cabal v2-update && cabal v2-build saw-remote-api
```
?

(I didn't write any of this code -- just `git mv`'d it around a little)